site/guide/generate-model-documentation.qmd

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
This file will be removed once some of the content has been re-homed or is reflected in how we structure our developer documentation.